apollo-router/src/services/router/service.rs

```diff
@@ -799,12 +801,18 @@ impl RouterService {
         for (code, count) in map {
             match code {
                 None => {
-                    tracing::info!(monotonic_counter.apollo.router.graphql_error = count,);
+                    u64_counter!(
+                        "apollo.router.graphql_error",
+                        "Number of GraphQL error responses returned by the router",
```

Comment:
Ah really? I thought we quite recently added more of these (though the metric name is definitely not good). Is there a replacement?

Reply:
```yaml
telemetry:
  instrumentation:
    instruments:
      router:
        http.server.request.duration:
          attributes:
            http.response.status_code: true
            graphql.operation.name:
              operation_name: string
            # This attribute will be set to true if the response contains graphql errors
            graphql.errors: # This will be true if it contains graphql errors
              on_graphql_error: true
```

Here is the alternative